### PR TITLE
View pricelists's prices using ngTable

### LINKED ISSUE #7: Show prices of a pricelist when viewing a specific pricelist

### PR DESCRIPTION
**Pull request by vritant**
@meghab2 take a look, and :+1: if you are okay with the merge, not sure if you have merge access.

fixes #7

### CODE REVIEW

**Comment by meghab2** [comment]
Hey.. did not work today... will be working on saturday.

Tomorrow will look into it

File: scripts/create_test_data.py [hunk 1]
```diff
@@ -107,12 +115,16 @@ def create_mill():
     print "Creating Mill: %s" % mill.name
     mill.code = get_code(mill.name)
     mill = mill_api.create_mill_using_post(mill)
+    labels = Set()
```
Comment (meghab2) [inline scripts/create_test_data.py:118]: any specific reason for it to assign as set

File: src/main/webapp/scripts/app/entities/priceList/priceList-detail.controller.js [hunk 2]
```diff
@@ -1,8 +1,74 @@
 'use strict';
 
 angular.module('omsApp')
-    .controller('PriceListDetailController', function ($scope, $rootScope, $stateParams, entity, PriceList, Price, Tax, CustomerGroup, DerivedGsmShade) {
+    .controller('PriceListDetailController', function ($scope, $rootScope, $stateParams, entity, PriceList, Price, Tax, CustomerGroup, DerivedGsmShade, ngTableParams, $filter) {
         $scope.priceList = entity;
+
+        function getUpperQuality(quality) {
+            return quality.replace(/\s+/g, '').toUpperCase();
+        }
```
Comment (meghab2) [inline src/main/webapp/scripts/app/entities/priceList/priceList-detail.controller.js:9]: :+1: Able to understand and review till here

File: src/test/java/com/vritant/oms/web/rest/PriceListResourceIntTest.java [hunk 3]
```diff
@@ -175,6 +225,27 @@ public void getPriceList() throws Exception {
...
+            .andExpect(content().contentType(MediaType.APPLICATION_JSON))
+            .andExpect(jsonPath("$.id").value(priceList.getId().intValue()))
+            .andExpect(jsonPath("$.pricess[0].value").value(2.3d))
+            .andExpect(jsonPath("$.pricess[0].mill.id").value(mill.getId().intValue()))
+            .andExpect(jsonPath("$.pricess[0].quality.id").value(q.getId().intValue()))
+            .andExpect(jsonPath("$.pricess[0].simpleGsmShade.id").value(sgs.getId().intValue()));
+    }
+
+    @Test
+    @Transactional
```
Comment (meghab2) [inline src/test/java/com/vritant/oms/web/rest/PriceListResourceIntTest.java:248]: Can you please brief me about this ?

File: src/main/webapp/scripts/app/entities/priceList/priceList-detail.controller.js [hunk 4]
```diff
@@ -1,8 +1,74 @@
...
+
+            var millPrices = sortPrices(priceList.pricess);
+
+            $scope.pricesTable = new ngTableParams({}, {
+                getData: function ($defer, params) {
+                    $scope.data = params.sorting() ? $filter('orderBy')(millPrices, params.orderBy()) : millPrices;
+                    $scope.data = params.filter() ? $filter('filter')($scope.data, params.filter()) : $scope.data;
+                    $defer.resolve($scope.data);
+                },
+                counts: []
```
Comment (meghab2) [inline src/main/webapp/scripts/app/entities/priceList/priceList-detail.controller.js:69]: This is the heart of Ngtable concept. will take some more time to understand this. Please merge this pull request

File: scripts/create_test_data.py (hunk 1 above)
Comment (vritant) [inline scripts/create_test_data.py:118]: to ensure mill has unique labels

File: src/test/java/com/vritant/oms/web/rest/PriceListResourceIntTest.java (hunk 3 above)
Comment (vritant) [inline src/test/java/com/vritant/oms/web/rest/PriceListResourceIntTest.java:248]: @Transactional ? basically runs the test in a transaction.
for more details http://docs.oracle.com/javaee/7/api/javax/transaction/Transactional.html

**Pull request merged**